pkg/test/framework/components/istioctl/kube.go

```diff
@@ -26,6 +27,12 @@ import (
 	"istio.io/istio/pkg/test/scopes"
 )
 
+var (
+	// We cannot invoke the istioctl library concurrently due to the number of global variables
+	// https://github.com/istio/istio/issues/37324
+	invokeMutex sync.Mutex
```

Comment:
at one point I thought we installed multi clusters in parallel. do we not do that already, or does this PR stop making that happen?

Reply:
Good point..

Reply:
@stevenctl  How is our installation safe if we're invoking istioctl concurrently?

Reply:
It's not.. but this part also doesn't seem to be horribly affected. 

We could also workaround by having a separate `Invoke` impl that execs the istioctl binary. Downside is things are harder to debug (we can control the impl with a flag?)